README.md

```diff
@@ -36,3 +37,5 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
+
+Test
```

Comment:
should we add instructions for running the tests under here?

Reply:
I think this was just a dangling text that has been there for a while.  I'll just remove it for now.  Good catch.